docs/modules/clusters/pages/ucn-dynamic-config.adoc

```diff
@@ -25,19 +25,22 @@ public class Client {
         HazelcastInstance instance = getHazelcastInstance();
         UserCodeNamespacesConfig parentConfig = instance.getConfig().getNamespacesConfig(); <1>
         UserCodeNamespaceConfig namespaceConfig = new UserCodeNamespaceConfig("my_namespace"); <2>
-        namespaceConfig.addClass(MyRequireClass.class); <3>
-        parentConfig.addNamespaceConfig(namespaceConfig); <4>
+            namespaceConfig.addClass(MyRequireClass.class); <3>
+            UserCodeNamespaceConfig ucn = new UserCodeNamespaceConfig("my_namespace")
```

Comment:
Programmatically this is fine. I'm just wondering, it looks a bit funny that nothing is ever done with this Object after it is created (from a demonstration pov). 

How about you add the class to the exist `namespaceConfig` you created on line 27? Just like you have done on line 28?